pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -84,42 +85,22 @@ workflow ImputationBeagle {
           gatk_docker = gatk_docker
       }
 
-      call tasks.CountVariantsInChunksBeagle {
+      call beagleTasks.CountVariantsInChunksBeagle {
```

Comment:
does it make sense to rename the tasks themselves to not include Beagle, since they're in the ImputationBeagleTasks wdl? or would that be more confusing because then there are two tasks with the same name (albeit coming from different source wdls)? I don't really have an opinion here

Reply:
yea I like removing `Beagle` from the names